cluster/validate-cluster.sh

```diff
@@ -52,7 +52,12 @@ CLUSTER_READY_ADDITIONAL_TIME_SECONDS="${CLUSTER_READY_ADDITIONAL_TIME_SECONDS:-
 
 EXPECTED_NUM_NODES="${NUM_NODES}"
 if [[ "${REGISTER_MASTER_KUBELET:-}" == "true" ]]; then
-  EXPECTED_NUM_NODES=$((EXPECTED_NUM_NODES+1))
+  if [[ "${KUBE_EXPERIMENTAL_REPLICATE_EXISTING_MASTER:-}" == "true" ]]; then
+    NUM_MASTERS=$(get-master-replicas-count)
```

Comment:
I think it will not work once we implement this "dynamic" clustering of apiservers as there will be no place with expected number of masters. Maybe we can just change this to verify that we have all the nodes and at least 1 master. What do you think?